Cargo.toml

```diff
@@ -59,6 +59,7 @@ tracing = { version = "0.1", features = ["log"] }
 thiserror = "1.0"
 png = "0.17"
 bitflags = "2.4"
+byteorder = "1.5"
```

Comment:
note: This is a legacy dependency, now that we have ReadCursor

Reply:
These are marked as deprecated in the follow up PR https://github.com/Devolutions/IronRDP/pull/419/commits/9503901f066b2341480c1a2b4a35357d8c262944.